website/upgrade-guides/0-12.html.markdown

```diff
@@ -211,7 +211,8 @@ in the usual way and apply it with Terraform 0.12.
 
 ### Remote state references
 
-The `terraform_remote_state` data source has changed slightly for the v0.12
+The `
```

Comment:
This bit looks like a mistake? I suspect this wasn't intended.

Reply:
@pselle thanks, fixed and rebased.